jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Query.java

```diff
@@ -116,11 +126,19 @@ public boolean isAllowDottedKeys() {
 		return allowDottedKeys;
 	}
 
+	public boolean isUseAllTypeNames() {
```

Comment:
Naming of this accessor seems to follow Java Bean conventions over English conventions. I suggest `isUsingAllTypeNames()` instead. Note that english is not my first language, you might have a better suggestion.